release-lib-master.Jenkinsfile

```diff
@@ -0,0 +1,189 @@
+pipeline {
+  agent {
+    kubernetes {
+      cloud params.JENKINS_AGENT_CLOUD_NAME
+      label "jenkins-slave-${UUID.randomUUID().toString()}"
+      serviceAccount "jenkins"
+      defaultContainer 'jnlp'
+      yaml """
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        labels:
+          app: "jenkins-${env.JOB_BASE_NAME}"
+          indy-pipeline-build-number: "${env.BUILD_NUMBER}"
+      spec:
+        containers:
+        - name: jnlp
+          image: quay.io/kaine/indy-stress-tester:latest
```

Comment:
We need to set this up in the factory2 namespace, probably, so we can get this stuff all consolidated in common infrastructure...in case anyone moves out of the team.

Reply:
Okay, I might create another image like: release-tool-box or so.
But I don't have access to factory2